app/views/fileStatus.scala.html

```diff
@@ -55,15 +55,12 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
                                         </h2>
                                         <div class="govuk-error-summary__body">
                                             <p class="govuk-body">@messages("filestatus.success.filecount", status.totalFiles)</p>
-                                            <p class="govuk-body">@messages("filestatus.success.review")</p>
+                                            <!-- <p class="govuk-body">@messages("filestatus.success.review")</p> -->
```

Comment:
Can this be deleted rather than just commented out?

It's fine for the prototype code to be a bit messy, but commented-out code makes it harder to read the rest of the code. And we can always use the git history if we ever want to put it back. :slightly_smiling_face: